src/__test__/generate-test-data.ts

```diff
@@ -20,8 +20,8 @@ export function generateLayout(
   nDims: number,
   nMeas: number,
   nRows: number,
-  qColumnOrder = [],
-  qGrandTotalRow = []
+  qColumnOrder = [] as number[],
+  qGrandTotalRow = [] as Record<string, string>[]
```

Comment:
This should work right?
```suggestion
  qColumnOrder: number[] = [],
  qGrandTotalRow:  Record<string, string>[] = []
```